packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,191 @@
+### Accordion
+
+The Accordion component is implemented using the [W3C Accordion pattern](https://www.w3.org/TR/wai-aria-practices/#accordion).
+
+#### Controlled Usage
+
+This advanced example demonstrates
+[controlled component](https://reactjs.org/docs/forms.html#controlled-components)
+usage. It also demonstrates additional header text and menu buttons in the `Accordion.Header`.
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { Toggle, Field, Label } = require('@zendeskgarden/react-forms/src');
+const { IconButton } = require('@zendeskgarden/react-buttons/src');
+const GearIcon = require('@zendeskgarden/svg-icons/src/16/gear-stroke.svg').default;
+const FolderIcon = require('@zendeskgarden/svg-icons/src/16/folder-open-stroke.svg').default;
+const { Dropdown, Menu, Item, Trigger } = require('@zendeskgarden/react-dropdowns/src');
+
+const StyledHint = styled.span`
```

Comment:
I'd be curious if we could advocate for using out-of-the-box component form Garden as opposed to styled component elements within the examples? At the very least, I believe UXCS advises against italics, but I wonder if we could just use typography components?

Reply:
Good call. The most recent typography components support this case – but @hzhu will need to release the updated v8.6.0 before they can be used here.

Reply:
👍 in the meantime, I've removed the italics.

Reply:
with the release of v8.6.0, this can just use the `<SM>` typography component (which `isBold={false}` by default).

Reply:
The text now uses `<SM>` from `react-typography`.